src/main/java/calculator/view/Input.java

```diff
@@ -0,0 +1,21 @@
+package calculator.view;
+
+
+public class Input {
+
+
+    public static String[] getInput(String inputString) throws IllegalArgumentException {
+        validate(inputString);
+        return splitFromInput(inputString);
```

Comment:
```suggestion
        return inputString.split(" ");
```
private 메소드를 사용해서 얻는 이점이 없어보입니다. String에서 제공하는 메소드를 바로 사용해도 되지 않나요?